src/com/hms_networks/americas/sc/package.html

```diff
@@ -1,8 +1,7 @@
 <HTML>
 <BODY>
-Utility classes to provide access to and management of the Ewon Flexy historical data system
-using EBDs (Export Block Descriptors).
-Utility classes to provide access to and management of the Ewon Flexy real data system.
+Utility classes to provide access to and management of the Ewon Flexy real data system and Ewon
```

Comment:
Is this supposed to say real-time data system?

Reply:
Yes -- fixed